lacework/resource_lacework_team_member.go

```diff
@@ -0,0 +1,557 @@
+		Create: resourceLaceworkTeamMemberCreate,
+		Read:   resourceLaceworkTeamMemberRead,
+		Update: resourceLaceworkTeamMemberUpdate,
+		Delete: resourceLaceworkTeamMemberDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: importLaceworkTeamMember,
+		},
+		Schema: map[string]*schema.Schema{
+			"email": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The email for the team member which will also be used as the username",
+			},
+			"first_name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The first name of the team member",
+			},
+			"last_name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The last name of the team member",
+			},
+			"company": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The company name",
+			},
+			"enabled": {
+				Type:        schema.TypeBool,
+				Default:     true,
+				Optional:    true,
+				Description: "The state of the team member, whether they are enabled or not",
+			},
+			"administrator": {
+				Type:          schema.TypeBool,
+				Default:       false,
+				Optional:      true,
+				ConflictsWith: []string{"organization"},
+				Description:   "Whether the team member has admin role access into the Lacework account",
+			},
+			"organization": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"administrator": {
+							Type:          schema.TypeBool,
+							Optional:      true,
+							Default:       false,
+							ConflictsWith: []string{"organization.0.user"},
+							Description:   "Whether the team member is an org level administrator",
+						},
+						"user": {
+							Type:          schema.TypeBool,
+							Optional:      true,
+							Default:       false,
+							ConflictsWith: []string{"organization.0.administrator"},
+							Description:   "Whether the team member is an org level user",
+						},
+						"admin_accounts": {
+							// We are not using Set because we need to use DiffSuppressFunc: https://github.com/hashicorp/terraform-plugin-sdk/issues/160
+							Type:             schema.TypeList,
+							Optional:         true,
+							Description:      "List of accounts the team member is an admin",
+							ConflictsWith:    []string{"organization.0.user", "organization.0.administrator"},
+							DiffSuppressFunc: diffCaseInsensitive,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+								StateFunc: func(val interface{}) string {
+									return strings.ToUpper(strings.TrimSpace(val.(string)))
+								},
+							},
+						},
+						"user_accounts": {
+							// We are not using Set because we need to use DiffSuppressFunc: https://github.com/hashicorp/terraform-plugin-sdk/issues/160
+							Type:             schema.TypeList,
+							Optional:         true,
+							Description:      "List of accounts the team member is a user",
```

Comment:
nit: I feel this description could be improved

```suggestion
							Description:      "List of accounts the team member belongs to as a user",
```